plugins/vst_base/CMakeLists.txt

```diff
@@ -3,7 +3,7 @@ IF(LMMS_SUPPORT_VST)
 INCLUDE(BuildPlugin)
 
 IF(LMMS_BUILD_WIN32)
-	ADD_DEFINITIONS(-DPTW32_STATIC_LIB)
+	ADD_DEFINITIONS(-DPTW32_STATIC_LIB -std=c++0x)
```

Comment:
> it's more like -std=c++11 I guess

Same effectiveness.  `c++0x` is fine for now.  We can switch it down the road as more compilers adopt the official standard.  BTW, this specific line superseded by https://github.com/LMMS/lmms/commit/607d3f4f6611b64d32ea170c2b60177785eaf8e2; Can be removed.